packages/roosterjs-editor-core/lib/corePlugins/ImageSelection.ts

```diff
@@ -70,19 +70,15 @@ export default class ImageSelection implements EditorPlugin {
                         !rawEvent.metaKey &&
                         keyDownSelection.type === SelectionRangeTypes.ImageSelection
                     ) {
-                        if (key === Escape) {
+                        const position = new Position(keyDownSelection.image, PositionType.Before);
+                        if (key === Escape && position) {
                             this.editor.select(keyDownSelection.image, PositionType.Before);
                             this.editor.getSelectionRange()?.collapse();
                             event.rawEvent.stopPropagation();
                         } else if (key === Delete) {
                             this.editor.deleteNode(keyDownSelection.image);
                             event.rawEvent.preventDefault();
-                        } else {
-                            const position = new Position(
-                                keyDownSelection.image,
-                                PositionType.Before
-                            );
-
+                        } else if (position) {
```

Comment:
actually position can never be null. The problem could be there is no parent element so that when we create Position, we stored null as node:

![image](https://github.com/microsoft/roosterjs/assets/23065085/cbccbe90-7f08-4b1e-9662-8565fb48d475)

Then in createRange, when we get nodeType, it throws:

![image](https://github.com/microsoft/roosterjs/assets/23065085/b9a85bfc-7bfd-4ae1-bf49-bc3b4b31f971)

To avoid this, we need to check if there is parent node of image. If not, it is impossible to select the image so we just do nothing.